server/liveblog/blogs/blogs.py

```diff
@@ -15,8 +15,9 @@
     'guid': metadata_schema['guid'],
     'title': metadata_schema['headline'],
     'description': metadata_schema['description'],
-    'language': Resource.rel('languages', True),
-    'theme': Resource.rel('themes', True),
+    'theme': {
```

Comment:
interesting and complicated, also I don't see the point, if relevant theme files could still change.